travel/templates/base.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+   <meta charset="utf-8">
+   <title>Principal Travel</title>
+   <link rel="stylesheet" href="/static/css/normalize.min.css">
+   <link rel="stylesheet" href="/static/css/google-fonts.css">
+    <link rel="stylesheet" href="/static/css/main.css">
+    <script src="static/js/vendor/jquery-1.11.3.min.js"></script>
+    <script src="static/js/components.js"></script>
```

Comment:
I didn't see this in Django documentations but added a head block inside the head tags. I hope that's what you had in mind.